### PR TITLE
docs: provide example regexes to properly escape . characters

### PR DESCRIPTION
**Pull request by bcpeinhardt**
Some of the example regular expressions we provide in the docs for managing coder external auth don't escape `.` characters correctly. This PR simply corrects that.

### CODE REVIEW

**Comment by bcpeinhardt** [comment]
@matifali added you as a reviewer so you'll have the context for the zendesk ticket on Monday.

**Comment by bcpeinhardt** [comment]
Do Not Merge until after docs restructure 🙏

File: docs/admin/external-auth.md
Comment (matifali) [inline docs/admin/external-auth.md:194]: Should we add a comment that it is only required if using multiple external auth for Github belonging to different organizations?

**Review by matifali — APPROVED** [review]
LGTM, Assuming you have already tested. Otherwise I will try to test later this week.

File: docs/admin/external-auth.md
Comment (bcpeinhardt) [reply docs/admin/external-auth.md:194]: I don't think so, we want the regex to be correct, and dropping the \ only opens the user up to the same issue if they do one day decide to do multiple external auth.

Comment (matifali) [reply docs/admin/external-auth.md:194]: I mean the `CODER_EXTERNAL_AUTH_0_REGEX` field it self. It is optional and only comes into action when someone have multiple GitHub auths.

**Comment by matifali** [comment]
@bcpeinhardt do you still want to merge this?

**Comment by bcpeinhardt** [comment]
Yes! It was failing CI before and I didn't have a chance to diagnose it.

**Pull request merged**
